README.md

```diff
@@ -1 +1,188 @@
-# sti-python
+Python for OpenShift - Docker images
+========================================
+
+This repository contains sources of the images for building various versions
+of Python applications as reproducible Docker images using
+[source-to-image](https://github.com/openshift/source-to-image).
+User can choose between RHEL and CentOS based builder images.
+The resulting image can be run using [Docker](http://docker.io).
+
+
+Versions
+---------------
+Python versions currently supported are:
+* python-3.3
+
+RHEL versions currently supported are:
+* RHEL7
+
+CentOS versions currently supported are:
+* CentOS7
+
+
+Installation
+---------------
+To build Python image, choose between CentOS or RHEL based image:
+*  **RHEL based image**
+
+    To build a rhel-based python-3.3 image, you need to run the build on properly
+    subscribed RHEL machine.
+
+    ```
+    $ git clone https://github.com/openshift/sti-python.git
+    $ cd sti-python
+    $ make build TARGET=rhel7 VERSION=3.3
+    ```
+
+*  **CentOS based image**
```

Comment:
we do, but for some reason its missing in ruby && nodejs, but it is in all other images
sti-base : https://github.com/openshift/sti-base#installation-and-usage
mongodb : https://github.com/openshift/mongodb#installation